### PR TITLE
R4 RelatedPerson Expose Extension relationship-level Read/Search

### PR DESCRIPTION
**Pull request by sydneanschutz**
Description
----
Created documentation for the relationship-level extension
Added extension to read/search examples (and narrative)

Screen Shot 2020-10-16 at 12 26 08 PM
Screen Shot 2020-10-16 at 2 04 55 PM

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

File: lib/resources/example_json/r4_examples_relatedperson.rb [hunk 1]
```diff
@@ -14,8 +14,24 @@ module Resources
...
       },
+      'extension': [
+        {
+          'url': 'https://fhir-ehr.cerner.com/r4/StructureDefinition/relationship-level',
+          'valueCodeableConcept': {
+            'coding': [
+              {
+                'system': 'http://hl7.org/fhir/resource-types',
+                'code': 'Patient',
+                'display': 'Patient Level Relationship'
```
Comment (kathypickering) [inline lib/resources/example_json/r4_examples_relatedperson.rb:28]: This should be the display from the system, thus needs to change to Patient

File: lib/resources/r4/related_person.yaml [hunk 2]
```diff
@@ -63,3 +63,16 @@ fields:
     - display: Millennium Person Relationships
       system: https://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/40
       info_link: https://fhir.cerner.com/millennium/r4/proprietary-codes-and-systems/#code-set-40-person-relationships
+
+- name: Relationship Level Extension
+  type: CodeableConcept
+  description: Reference to the Encounter associated to the encounter level RelatedPerson.
```
Comment (kathypickering) [inline lib/resources/r4/related_person.yaml:69]: I don't think I've ever reviewed the yaml file.  Why aren't you using the Description from the Structure Definition?
Such as The resource's relationship to either the patient or encounter level.
Same comment for the binding description

File: lib/resources/example_json/r4_examples_relatedperson.rb (hunk 1 above)
Comment (sydneanschutz) [reply lib/resources/example_json/r4_examples_relatedperson.rb]: Fixed: https://github.com/cerner/fhir.cerner.com/pull/480/commits/98af69a03800c29aa7b196f80be2dd2bd7ab0ee1#diff-133ac83b73a8a88b430a777cf1700b9010258bd96bc4ff69a51350091f73643eR28
Also fixed it in the narratives as well

File: lib/resources/r4/related_person.yaml (hunk 2 above)
Comment (sydneanschutz) [reply lib/resources/r4/related_person.yaml]: Copied from the wrong structure def! 
Fixed: https://github.com/cerner/fhir.cerner.com/pull/480/commits/98af69a03800c29aa7b196f80be2dd2bd7ab0ee1#diff-261586474bd9faacdccf38c13eea61175a7bba5b7ce8ddb4dea273142eb9f6d6L69

**Review by kathypickering — APPROVED** [review]

**Review by exopsRamba — APPROVED** [review]

**Review by MadhurRaj — CHANGES_REQUESTED** [review]

File: content/millennium/r4/individuals/related-person.md [hunk 3]
```diff
@@ -105,3 +117,4 @@ The common [errors] and [OperationOutcomes] may be returned.
 [`token`]: http://hl7.org/fhir/R4/search.html#token
```
[the remark this answers is not in the document]
Comment (sydneanschutz) [reply content/millennium/r4/individuals/related-person.md]: https: https://github.com/cerner/fhir.cerner.com/pull/480/commits/c72ab2b59e1405a76fe9f672ff82ecb9ff419737

File: content/millennium/r4/individuals/related-person.md [hunk 4]
```diff
@@ -31,6 +31,18 @@ The following fields are returned if valued:
...
+
+* [Relationship Level]
+
+### Custom Extensions
+
+All URLs for custom extensions are defined as `https://fhir-ehr.cerner.com/r4/StructureDefinition/{id}`
+
+ ID                   | Value\[x] Type                                                             | Description
+----------------------|----------------------------------------------------------------------------|----------------------------------------------------------------
+ `relationship-level` | [`CodeableConcept`](http://hl7.org/fhir/r4/datatypes.html#CodeableConcept) | The resource's relationship to either patient or encounter.
```
[the remark this answers is not in the document]
Comment (sydneanschutz) [reply content/millennium/r4/individuals/related-person.md]: inline: https://github.com/cerner/fhir.cerner.com/pull/480/commits/b5baa8e9908086943176120b6f35afb04cf8c2f1

https: https://github.com/cerner/fhir.cerner.com/commit/c72ab2b59e1405a76fe9f672ff82ecb9ff419737

**Review by MadhurRaj — APPROVED** [review]

**Comment by DanielWhiteCerner** [comment]
image

**Pull request merged**